runtime/drivers/duckdb/connectors.go

```diff
@@ -118,6 +121,86 @@ func (c *connection) ingest(ctx context.Context, env *connectors.Env, source *co
 	return summary, nil
 }
 
+func (c *connection) ingestMotherduckData(ctx context.Context, source *connectors.Source, env *connectors.Env) (*drivers.IngestionSummary, error) {
+	err := c.WithConnection(ctx, 1, func(ctx, ensuredCtx context.Context) error {
+		res, err := c.Execute(ctx, &drivers.Statement{Query: "SELECT current_database();"})
+		if err != nil {
+			return c.checkErr(err)
+		}
+		defer res.Close()
+
+		res.Next()
+		var localDB string
+		if err := res.Scan(&localDB); err != nil {
+			return c.checkErr(err)
+		}
+
+		// get token
+		token := env.Variables["TOKEN"]
+		if token == "" && env.AllowHostAccess {
+			token = os.Getenv("motherduck_token")
+		}
+		// load motherduck extension; connect to motherduck service
+		err = c.Exec(ctx, &drivers.Statement{Query: "LOAD 'motherduck';", Priority: 1})
```

Comment:
Does this need to be `INSTALL 'motherduck'; LOAD 'motherduck';`?

Reply:
Also, you can omit `Priority` from all these queries – when using `WithConnection`, a conn is already acquired, so priority doesn't matter

Reply:
> Does this need to be INSTALL 'motherduck'; LOAD 'motherduck';?

Yes it worked for me since I already had installed `motherduck`. Wouldn't work on new machine. Will make the change.